DEVELOPMENT.md

```diff
@@ -0,0 +1,75 @@
+# Development
+
+This contains some notes on developing this software locally.
+
+# prerequisites
+
+* [goreleaser](http://goreleaser.com)
+* [go 1.24](https://go.dev)
+
+# building
+
+## Local Build
+
+Build the binary locally.
+
+```bash
+make build
```

Comment:
Could we add a `make install` which `go install`'d it into `${GOPATH}/bin`?

Reply:
Yeah for sure, I normally use a local binary in the project directory for opensource projects where devs from a wide array of backgrounds are contributing, local directory is easier to understand.

Reply:
yeah that's fair!